R/qenv-class.R

```diff
@@ -4,6 +4,15 @@
 #' @name qenv-class
 #' @rdname qenv-class
 #' @slot code (`expression`) to reproduce the environment
+#' @slot code_dependency A `list` equal to the length of `id`, where each element is a list containing 3 elements
+#' - `occurrence` - named `list` by object names with numeric vector as elements indicating calls in which object
+#' appears.
+#' - `cooccurrence` - `list` of the same length as number of calls in `parsed_code`, containing `NULL`s if there is no
+#' co-occurrence between objects, or a `character` vector indicating co-occurrence of objects in specific `parsed_code`
+#' call element. If a character vector, then the first element is the name of the dependent object, and the rest are the
+#' influencing objects
+#' - `effects` - named `list`  by object names with numeric vector as elements indicating which calls has effect on this
+#' object, or NULL if there are no side-effects pointing at this object.
```

Comment:
got it, thanks for the feedback, will try to get this improved

Reply:
Hey, I updated the phrasing in here e4b6ba6cb2085b01fddedd456e5c978cc0368022
`parsed_code` is explained since it is an input parameter and explained in `@param` field. + this is explained in `@description`